### ISSUE TITLE
Expose accent color config

### ISSUE THREAD

**Issue by jakewies** [jakewies/hugo-theme-codex#51]
The current color scheme revolves around a single primary color: `#9013fe`. I think we could make this a configurable value, but I'm not yet sure how we can access the value in a stylesheet. Just something that popped into my head. I like the idea of having a configurable accent color that is used throughout the site.

**Comment by kentnek** [comment]
Yep this is what's on my mind for the last few days. My previous SCSS PR did make it possible to use Hugo variables in scss files, but unfortunately it doesn't work on partials like `_colors.scss` or `_vars.scss`. There are a few workarounds, but the main gist is to move all configurable scss variables to a separate file (and remove existing hard-coded values along the way).

I was looking at this [theme](https://github.com/bep/docuapi) to see how they accomplished it. I could take this task too if you don't mind.

**Comment by jakewies** [comment]
Sure thing go for it! And ya I agree, TOC should come first. Just had it on my mind and wanted to dump it before I forget.

**Comment by kentnek** [comment]
Hmm actually I was thinking of moving all configurable/overridable variables into a separate scss file, something like [this](https://github.com/bep/docuapi/blob/master/assets/scss/slate/_variables.scss). Then the user can override this file with a dedicated overriding scss (like [this](https://github.com/bep/docuapi/blob/master/assets/scss/slate/docuapi_overrides.scss)) as necessary.

Gotemplate variables are already supported in the `customs.scss` file, but since it's technically still SCSS, I haven't found a way to suppress my IDE's nags either.

**Comment by jakewies** [comment]
Ok that's a good idea though. If we add the accent color is a variable in a scss file that is documented as being "overrideable", then we shouldn't have to pull value from `config.toml`, and we won't have to deal with IDE's yelling at us.

Should we add this value to `custom.scss`.

**Comment by kentnek** [comment]
So IMO `custom.scss` is to override the overall styles (as it's applied last), while the `overrides.scss` should be applied right after the `_variables.scss`, and before any other styles. 

In the `docuapi` theme, I think the line `@import overrides.scss` should have been placed at the end instead (to override variables defined before the import), but I'm not entirely sure if the import order matters either. Could you help me test that?

**Comment by jakewies** [comment]
Sure I'll take a peak. I'll probably open a PR to handle this and we can move discuss into there.
